WTWSMS/events/job_lord_spiritual.txt

```diff
@@ -2757,7 +2757,7 @@ character_event = {
 							text = WAGING_CRUSADE
 							hidden_tooltip = {
 								using_cb = crusade
-								using_cb = religious
+								#using_cb = religious
```

Comment:
Why did you disable this and the below? Isn't it just missing an "OR"?

Reply:
two instances of using_cb (look at line 2759)

Reply:
I can add "OR" and re-enable the second CB.

Reply:
Done

Reply:
Yes, because OR is definitely what is intended.